src/ampAphroditeInterfaceFactory.js

```diff
@@ -20,18 +19,14 @@ const cssArgNormalizer = (arg, create) => {
   return create({ [INLINE_STYLE_KEY]: arg })[INLINE_STYLE_KEY];
 };
 
-function withAmp(styles, resolve, create) {
-  if (isAmp()) {
-    return {
-      className: injectAndGetClassName(
-        false,
-        styles.map(cullResponsiveStylesForAmp).map(arg => cssArgNormalizer(arg, create)),
-        defaultSelectorHandlers,
-      ),
-    };
-  }
-
-  return resolve(styles);
+function withAmp(styles, create) {
```

Comment:
I was confused since this is named like an HOC, can we rename it to something like `getResolvedStyles`?